### DISCUSSION TITLE
How to configure directus to use Google Drive Storage?

### DISCUSSION THREAD

**Discussion by IlyaPetrovM** [directus/directus#7588] [Archive]
Hi!
How to set google drive storage for my server? 
Am I right that I have to write `STORAGE_AWS_DRIVER="google"` and
place KEY and SECRET to `.env` file?

Where I can find this data? 

```STORAGE_LOCATIONS="aws"

STORAGE_AWS_DRIVER="s3"
STORAGE_AWS_KEY="tp15c...510vk"
STORAGE_AWS_SECRET="yk29b...b932n"
STORAGE_AWS_REGION="us-east-2"
STORAGE_AWS_BUCKET="my-files"
```

**Comment by rijkvanzanten** [comment]
The configuration you shared is for S3(-compatible) services specifically. Use the following keys for Google Cloud Storage:

https://docs.directus.io/reference/environment-variables/#google-cloud-storage-gcs

Also, be aware that we support Google _Cloud Storage_, not Google Drive

**Reply by IlyaPetrovM** [reply]
Aah! I supposed that was Google Drive... ok.

**Comment by ipcam4g** [comment]
Hello, do you already have official support for Google Drive?

**Reply by rijkvanzanten** [reply]
There's official support for Google Cloud Storage, but not Google Drive. There's also no open feature request for that, so feel free to create one 👍🏻

**Reply by ipcam4g** [reply]
@rijkvanzanten 
@stevefan1999-personal 

Hello, I have opened this request for a new feature.

https://github.com/directus/directus/discussions/16124

**Comment by stevefan1999-personal** [comment]
you would better wait for https://github.com/rclone/rclone/pull/6461
